pkg/trace/stats/concentrator.go

```diff
@@ -48,6 +49,27 @@ type Concentrator struct {
 	agentVersion           string
 	peerSvcAggregation     bool // flag to enable peer.service aggregation
 	computeStatsBySpanKind bool // flag to enable computation of stats through checking the span.kind field
+	extraTags              []string
+}
+
+// TODO: finish full list
+// PeerServiceEntityTags represents the full list of supplementary tags to describe a peer.service entity.
+var PeerServiceEntityTags = []string{"rpc.host", "grpc.host", "db.instance", "db.system", "kafka.topic"}
```

Comment:
@djova I worry a little bit about the cost if this slice grows to become 20+ elements. We might need to get smarter over time, like an index of `op_name: [entity_tags]`.

Reply:
Let's try to be as smart as possible from the jump so that we don't have to check these additional tags for every single span we see.